src/js/App/Feedback/Feedback.js

```diff
@@ -29,7 +37,7 @@ const Feedback = ({ user }) => {
         },
         body: JSON.stringify({
           description: `Feedback: ${textAreaValue}, Username: ${user.identity.user.username}, Account ID: ${user.identity.account_number}, Email: ${user.identity.user.email}, URL: ${window.location.href}`, //eslint-disable-line
-          summary: `${window.insights.chrome.isProd ? '[PROD]' : '[PRE-PROD]'} App Feedback`,
+          summary: `${addFeedbackTag()} App Feedback`,
```

Comment:
can we put the brackets around this so it comes up as `[Insights] App Feedback`?

Reply:
oh good catch